### ISSUE TITLE
Replace "disable-ssl-redirect" annotation with "ssl-redirect" 

### ISSUE THREAD

**Issue by nilathedragon** [caddyserver/ingress#102]
In NGINX ssl-redirect is controlled by the "ssl-redirect" annotation.
I saw you implemented this functionality by it being enabled by default and the annotation "disable-ssl-redirect" to turn it off. 

I think it would make more sense to have naming parity with the nginx annotation. It makes more sense to explicitly turn "ssl-redirect" off than turning on a disable switch which then turns off ssl-redirect.

The default for the "ssl-redirect" annotation would be "true" to match the current behaviour.

Example NGINX:
```
nginx.ingress.kubernetes.io/ssl-redirect: "false"
```

Example Caddy (Current)
```
caddy.ingress.kubernetes.io/disable-ssl-redirect: "true"
```

Example Caddy (Proposed)
```
caddy.ingress.kubernetes.io/ssl-redirect: "false"
```

Priority wise it would make sense to have ssl-redirect > disable-ssl-redirect so there is still support for the old annotation?

**Comment by Embraser01** [comment]
Yep, I'd be ok with that. I agree having an annotation starting with `disable-` is quite confusing, having a default value to `true` is better (it's not documented though, but that should change as soon as I finished the documentation)
